src/Promitor.Core/EnvironmentVariables.cs

```diff
@@ -11,6 +11,7 @@ public class Authentication
         {
             public const string ApplicationId = "AUTH_APPID";
             public const string ApplicationKey = "AUTH_APPKEY";
+            public const string ManagedIdentityId = "AUTH_UAMI";
```

Comment:
Let's use this to be more clear?
```suggestion
            public const string IdentityId = "AUTH_MANAGED_IDENTITY_ID";
```

Reply:
Well, UAMI is the acronym used everywhere (you will often find it on stackoverflow for instance) when we are referencing a User Assigned Managed Identity.
If you want to have a full name identifier, I would replace it with "AUTH_USER_ASSIGNED_MANAGED_IDENTITY_ID"
Choose the one you prefer :)

Reply:
I know but it's a bit too long so I went for this, don't want to overcomplicate it :D